### ISSUE TITLE
Build on arm64

### ISSUE THREAD

**Issue by vinaooo** [rom1v/autoadb#10]
Is it possible?

**Comment by rom1v** [comment]
Yes, this is basically the same as for cross-compiling gnirehtet for arm: https://github.com/Genymobile/gnirehtet/issues/46#issuecomment-335382429

EDIT: oh, if you want to build ON arm64, just build on a arm64 machine.
